src/system/TLVPacketBufferBackingStore.h

```diff
@@ -80,6 +80,11 @@ class TLVPacketBufferBackingStore : public chip::TLV::TLVBackingStore
     CHIP_ERROR OnInit(chip::TLV::TLVWriter & writer, uint8_t *& bufStart, uint32_t & bufLen) override;
     CHIP_ERROR GetNewBuffer(chip::TLV::TLVWriter & writer, uint8_t *& bufStart, uint32_t & bufLen) override;
     CHIP_ERROR FinalizeBuffer(chip::TLV::TLVWriter & writer, uint8_t * bufStart, uint32_t bufLen) override;
+    virtual bool IsSafeToReserve() override
+    {
+        // It is safe to reserve if there is no chained buffers.
```

Comment:
This re-states the code. Maybe explaining why this is the case would be better, for the comment to add to it rather than re-state.